python/lsst/ts/idl/enums/TunableLaser.py

```diff
@@ -20,6 +20,8 @@ class LaserDetailedState(enum.IntEnum):
 
     NONPROPAGATING = 1
     PROPAGATING = 2
+    PROPAGATINGBURSTMODEWAITINGFORTRIGGER = 3
+    PROPAGATINGBURSTMODETRIGGERED = 4
```

Comment:
Please consider underscores to make this more readable, e.g. PROPAGATING_BURST_MODE_WAITING_FOR_TRIGGER

Reply:
Please add a description of these two new modes to the doc string above